arch/aarch64/cpu.c

```diff
@@ -1,5 +1,11 @@
 #include <kernel/cpu.h>
 
+//////////////////////////////////////////////////////////////////////////
+// \brief the function initializes and asm volatile memory
```

Comment:
This is incorrect. The `arch_cpu_halt` function halts the CPU and makes it wait for an interrupt to wake it up. See the Intel manuals on `hlt` instruction or AArch64 manuals for `wfi` instruction on more information.

Reply:
🆗